### ISSUE TITLE
import problems

### ISSUE THREAD

**Issue by jjalexander** [Zij-IT/EnumCycling#1]
I had to add both "use enum_cycling::EnumCycle;" and "use enum_cycling_derive::EnumCycle;" in order to get the "#[derive(EnumCycle)]" code to work. The second use statement is not in the readme.

Is this normal?

**Comment by Zij-IT** [comment]
That is definitely not what I am expecting. Ideally you would be able to import the `EnumCycle` derive with just

```rust
use enum_cycling::EnumCycle;
```
while using the derive feature as such:

```toml
enum_cycling = { version = "0.1.1", feature = "derive" }
```
The goal is to have the same behaviour as both Serde and Strum. I will look into it, and hopefully have an answer by the end of the coming week!

**Comment by jjalexander** [comment]
Also, the dependencies had to be added separately in the .toml file.

**Comment by tetrogem** [comment]
I believe the reason this is happening is because the 0.1.1 release of enum_cycling on crates.io still has its Cargo.toml set up to depend on 0.1.0 of enum_cycling_derive for some reason, which tries to impl IntoEnumCycle, which doesn't seem to exist anymore. Manually adding enum_cycling_derive adds version 0.2.0 which impls EnumCycle instead, which fixes the issue

**Comment by Zij-IT** [comment]
Better late than never :sweat: It should be fixed in `2.1` so now the only steps required to use it properly are:

1. `cargo add enum_cycling --features derive`
2. `use enum_cycling::EnumCycle`

@TetroGem Thank you for the advice! The issue actually ended up being that `enum_cycling_derive` didn't have `enum_cycling` as a dev dependency, and therefore couldn't actually derive the trait. I based the structure off of Serde and I must have missed that `serde_derive` uses `serde` as a dev dependency
